accelerator/config.h

```diff
@@ -188,6 +190,13 @@ status_t ta_core_set(ta_core_t* const core);
  */
 void ta_core_destroy(ta_core_t* const core);
 
+/**
+ * @brief Switch logger verbose level between quiet and verbose
+ * @param quiet[in] Switch to quiet mode or not
```

Comment:
Reformat this as `@param[in] quiet` thx 
I was wrong. The format above is the correct doxygen syntax